components/server/src/projects/projects-service.ts

```diff
@@ -115,7 +115,7 @@ export class ProjectsService {
     protected async onDidCreateProject(project: Project, installer: User) {
         let { userId, teamId, cloneUrl } = project;
         const parsedUrl = RepoURL.parseRepoUrl(project.cloneUrl);
-        if ("gitlab.com" === parsedUrl?.host) {
+        if ("gitlab.com" === parsedUrl?.host || "bitbucket.org" === parsedUrl?.host) {
```

Comment:
**issue(non-blocking)**: Prebuilds are still not enabled for the default branch for GitLab and Bitbucket repositories but certainly out of the scope of this PR. ➿

Reply:
That's interesting. Is this already planned/discussed/tracked somewhere in a GH issue?

Reply:
@AlexTugarev I could be missing something but I've added https://github.com/gitpod-io/gitpod/issues/7424 to continue the discussion about this issue there. Up for closing it if the issue is not accurate or valid. 🏀